docs/pricing/orders.mdx

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 1
+---
+
+# Orders
+
+An Order is a collection of Products and Prices that can be purchased by a Customer. Orders can be created manually on the platform our through automations that run upon certain events, such as a Journey submission.
+
+Our platform is build in a way such that [Business Entities](/docs/entities/flexible-entities) can be installed by our customers, and on this context we can also claim that we "eat our own dog food". This means that we use our own platform to build and model the Order Entity.
+
+<img style={{ boxShadow: '#a2a2a2 0px 0px 25px -8px', borderRadius: '5px' }} src="/img/order-detailsview.png" alt="Order Details View" />
+
+## Managing Orders via Pricing API
+
+Using our [Entity API](/docs/entities/entity-api) we can directly manage Orders, although very rapidly we realise that an Order has more to it than just a collection of Products and Prices. We needed to be able to manage the Order's lifecycle, and we needed to be able to manage the Order's state. We also needed to be able to recompute the order's product and price totals, along with the order price and product relations. Using just our Entity API would not suffice here since there is additional logic required to fully manage an order.
+
+To fullfil this purpose we have designed the [Pricing API](/docs/pricing/pricing-api), which is a wrapper around the Entity API. The Pricing API allows us to manage Orders, and it also allows us to manage the Order's lifecycle and state. The Pricing API also allows us to recompute the order's product and price totals, along with the order price and product relations.
+
+
+### Updating an Order using `epilot-sdk`
```

Comment:
Creating Orders via the API is the most common use case, so I’d consider using that as the example here.

Reply:
I've added a way to provide both examples with an API example and an SDK example.

![Screen Shot 2022-09-20 at 17 04 25](https://user-images.githubusercontent.com/4655079/191308767-77b073de-2f6b-4f59-8fd7-acf40b2cf52e.png)